test/functionality/dataflow/processing-of-elements/functions/function-call-tests.ts

```diff
@@ -310,9 +310,9 @@ a()()`,
 		assertDataflow('Calling with 1 + x', shell, 'foo(1 + x)',
 			new DataflowGraph()
 				.addVertex({ tag: 'function-call', id: '5', name: 'foo', environment: initializeCleanEnvironments(), args: [{ nodeId: '4', name: `${UnnamedArgumentPrefix}4`, scope: LocalScope, used: 'always' }]})
-				.addVertex({ tag: 'use', id: '4', name: `${UnnamedArgumentPrefix}4`})
-				.addVertex({ tag: 'use', id: '2', name: 'x'})
-				.addEdge('4', '2', EdgeType.Reads, 'always')
+				.uses('4', `${UnnamedArgumentPrefix}4`)
```

Comment:
maybe there is even a way to avoid writing all of these unnamed argument prefixes :eyes:.